DISCOVER/06_childcare.md

```diff
@@ -24,7 +24,7 @@
  
 - There are companies that provide childcare at conferences. While NumFOCUS makes no specific endorsements, we did find these companies in our research who are providing event-based childcare:
    - [Corporate Kids Events](https://conferencechildcare.com/)
-   - [American Child Care, Inc.](http://www.americanchildcare.com/conventions.htm)
+   - [American Child Care, Inc.](https://www.americanchildinc.org)
```

Comment:
We should just remove this link. That link doesn't go to a page talking about providing child care at conferences, but rather a single child care center.